user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -117,3 +117,30 @@ subforum for an up-to-date list!
 - Make any such classes where you like, and add appropriate
   service methods in ``app/Config/Services.php`` to load
   your components instead of the default ones
+
+Upgrading Libraries
+============================================================
+
+
+- Your app classes can still go inside app/Libraries, but they don’t have to
+- Instead of CI3’s ``$this->load->library(x);`` you can now use ``$this->x = new X();``,
+  following namespaced conventions for your component
```

Comment:
```suggestion
- Your app classes can still go inside ``app/Libraries``, but they don’t have to.
- Instead of CI3’s ``$this->load->library(x);`` you can now use ``$this->x = new X();``,
  following namespaced conventions for your component
```